src/responder/common/responder_get_domains.c

```diff
@@ -224,6 +224,26 @@ struct tevent_req *sss_dp_get_domains_send(TALLOC_CTX *mem_ctx,
     return req;
 }
 
+static void sss_resp_update_certmaps(struct resp_ctx *rctx)
+{
+    int ret;
+    struct certmap_info **certmaps;
+    bool user_name_hint;
+    struct sss_domain_info *dom;
+
+    for (dom = rctx->domains; dom != NULL; dom = dom->next) {
```

Comment:
No, just the domains are ok because the certmaps include the information for which domain they are valid.